test/steps/utils.js

```diff
@@ -79,6 +80,7 @@ export const getActionCreator = actionName => ({
 	'encrypt message': encryptMessage.actionCreator,
 	'encrypt passphrase': encryptPassphrase.actionCreator,
 	'create transaction register delegate': createTransactionRegisterDelegate.actionCreator,
+	'create transaction cast vote': createTransactionCastVote.actionCreator,
```

Comment:
This should match our canonical form (`create transaction cast votes`), as should the file name and the import etc. (Or the documentation should match this form if this is the canonical one.)